client/src/components/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useCallback } from "react";
+
+import enTranslations from "@shopify/polaris/locales/en.json";
+import { AppProvider, Page } from "@shopify/polaris";
+import { NewPageForm } from "./NewSurveyForm";
+import { NewSurveyModal } from "./NewSurveyModal";
+import { SurveyList } from "./SurveyList";
+
+export function App() {
+  const [openModal, setOpenModal] = useState(false);
+  const toggleModalHandler = useCallback(() => {
+    setOpenModal(!openModal);
+  }, [openModal]);
+
+  return (
+    <AppProvider i18n={enTranslations}>
```

Comment:
do we really need the translations tho

Reply:
It's required to get a barebones polaris app, but we don't need to leverage this. It's just boilerplate.